### ISSUE TITLE
How can I contribute translation?

### ISSUE THREAD

**Issue by Kifranei** [vfsfitvnm/ViMusic#478]
I want to contribute the translation of simplified Chinese, but I can't find the string.xml file

**Comment by vfsfitvnm** [comment]
Yep, tracking in #100. I will add translations once the development rate will be steady to a significantly lower pace.
